autoinstall/Pardus/preseed.cfg

```diff
@@ -0,0 +1,250 @@
+# installed, and what software you use. The default is not to report back,
+# but sending reports helps the project determine what software is most
+# popular and should be included on the first CD/DVD.
+popularity-contest popularity-contest/participate boolean false
+
+### Boot loader installation
+# Grub is the boot loader (for x86).
+
+# This is fairly safe to set, it makes grub install automatically to the UEFI
+# partition/boot record if no other operating system is detected on the machine.
+d-i grub-installer/only_debian boolean true
+
+# Due notably to potential USB sticks, the location of the primary drive can
+# not be determined safely in general, so this needs to be specified:
+#d-i grub-installer/bootdev string /dev/sda
+# To install to the primary device (assuming it is not a USB stick):
+d-i grub-installer/bootdev string default
+
+# Avoid that last message about the install being complete.
+d-i finish-install/reboot_in_progress note
+
+# This will prevent the installer from ejecting the CD during the reboot,
+# which is useful in some situations.
+#d-i cdrom-detect/eject boolean true
+
+### Preseeding other packages
+# Depending on what software you choose to install, or if things go wrong
+# during the installation process, it's possible that other questions may
+# be asked. You can preseed those too, of course. To get a list of every
+# possible question that could be asked during an install, do an
+# installation, and then run these commands:
+   debconf-get-selections --installer > /target/root/preceed.cfg
+   debconf-get-selections >> /target/root/preceed.cfg
+
+# This command is run just before the install finishes, but when there is
+# still a usable /target directory. You can chroot to /target and use it
+# directly, or use the apt-install and in-target commands to easily install
+# packages and run commands in the target system.
+d-i preseed/late_command string \
+    echo "Check network ..." > /target/dev/ttyS0; \
+    ip addr show > /target/dev/ttyS0; \
+    echo "Linkup network ..." > /target/dev/ttyS0; \
+    chroot /target /sbin/ifconfig $(/sbin/ifconfig -a | grep '^[a-z]' | cut -d: -f1 | head -n 1) up; \
+    sleep 5; \
+    echo "Check network after linkup ..." > /target/dev/ttyS0; \
+    ip addr show > /target/dev/ttyS0; \
+    echo "Check file /etc/apt/sources.list ..." > /target/dev/ttyS0; \
+    cat /target/etc/apt/sources.list > /target/dev/ttyS0; \
+    echo "Install packages ..." > /target/dev/ttyS0; \
+    chroot /target apt-get update -y > /dev/ttyS0; \
+    chroot /target apt-get install -y --no-upgrade openssh-server > /dev/ttyS0; \
+    chroot /target apt-get install -y --force-yes build-essential open-vm-tools sg3-utils vim > /dev/ttyS0; \
+    echo "Install package open-vm-tools-desktop if has desktop environment ..." > /target/dev/ttyS0; \
+    if [ -f "/target/usr/bin/Xorg" ]; then chroot /target apt-get install -y open-vm-tools-desktop > /target/dev/ttyS0; fi; \
```

Comment:
You can echo the message "Install package open-vm-tools-desktop" in this if.

Reply:
Thanks. Done